packages/blade/src/components/Icons/iconMap.ts

```diff
@@ -346,11 +342,11 @@ const iconMap: Record<string, IconComponent> = {
   PlusCircleIcon: PlusCircleIconComponent,
   PlusSquareIcon: PlusSquareIconComponent,
   PowerIcon: PowerIconComponent,
-  RefreshCwIcon: RefreshCwIconComponent,
-  RefreshCcwIcon: RefreshCcwIconComponent,
+  RefreshLeftIcon: RefreshLeftIconComponent,
+  RefreshRightIcon: RefreshRightIconComponent,
   RepeatIcon: RepeatIconComponent,
-  RotateCwIcon: RotateCwIconComponent,
-  RotateCcwIcon: RotateCcwIconComponent,
+  RotateClockWiseIcon: RotateClockWiseIconComponent,
+  RotateCounterClockWiseIcon: RotateCounterClockWiseIconComponent,
```

Comment:
Should we check the usage of RefreshLeftIcon & maybe do a breaking change for it? 

It doesn't feel very consistent to name the `Rotate` icons with clockWise convention & `Refresh` icon left/right.